gossip/service.go

```diff
@@ -127,12 +128,20 @@ func (f *ServiceFeed) Start(store ArchiveBlockHeightSource) {
 
 			height, empty, err := store.GetArchiveBlockHeight()
 			if err != nil {
-				log.Error("failed to get archive block height", "err", err)
-				continue
-			}
-			if empty {
-				continue
+				// If there is no archive, set height to the last block
+				// and send all notifications
+				if err == evmstore.NoArchiveError {
```

Comment:
nit: To be resilient towards future changes, this should be
```suggestion
				if errors.Is(err, evmstore.NoArchiveError) {
```